packages/ui/src/assets/css/stepper.css

```
@@ -0,0 +1,96 @@
+:root {
+  --step-completed-color: #007aff;
+  --step-active-border-color: #007aff;
+  --line-border-color: #e5e7eb;
```

Not sure why this doesn't have --step prefix.
```suggestion
  --step-line-border-color: #e5e7eb;
```

---

Also, I think the prefix should be `--stepper`. We will not use the Step component directly, so stepper makes more sense.